nano/node/blockprocessor.cpp

```diff
@@ -392,10 +388,6 @@ nano::process_return nano::block_processor::process_one (nano::write_transaction
 				node.logger.try_log (boost::str (boost::format ("Gap source for: %1%") % hash.to_string ()));
 			}
 			info_a.verified = result.verified;
-			if (info_a.modified == 0)
```

Comment:
Same as stated on the another PR (https://github.com/nanocurrency/nano-node/pull/3728). It seems we need an assert here so we ensure the `modified` field won't pass without being set.

Reply:
Fixed in the other PR.